curl/src/main/scala/org/http4s/curl/CurlClient.scala

```diff
@@ -62,7 +62,7 @@ private[curl] object CurlClient {
           IO(gcr.clear())
         }
 
-        dispatcher <- Dispatcher[IO]
+        dispatcher <- Dispatcher.parallel[IO]
```

Comment:
With some redesign, it might be possible to use the new `sequential` Dispatcher instead, for better performance. But as written today I believe it needs a `parallel` dispatcher.

Reply:
- https://github.com/http4s/http4s-curl/issues/53